chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md

```diff
@@ -195,7 +195,7 @@ $$\begin{aligned}f(u, v) & = (u+v)^{2} \\u(a, b) & = (a+b)^{2}, \qquad v(a, b) =
 
 Sinir ağları ile çalışırken bu tür denklem zincirleri yaygındır, bu nedenle bu tür işlevlerin gradyanlarının nasıl hesaplanacağını anlamaya çalışmak çok önemlidir. Hangi değişkenlerin doğrudan birbiriyle ilişkili olduğuna bakarsak, bu bağlantının görsel ipuçlarını :numref:`fig_chain-1`'de görmeye başlayabiliriz.
 
-![Düğümlerin değerleri temsil ettiği ve kenarların işlevsel bağımlılığı gösterdiği yukarıda geçen işlev ilişkileri.](../img/ChainNet1.svg)
+![Düğümlerin değerleri temsil ettiği ve kenarların işlevsel bağımlılığı gösterdiği yukarıda geçen işlev ilişkileri.](../img/chain-net1.svg)
```

Comment:
This svg image is not present. In the english repo it got renamed a while back. See [commit](https://github.com/d2l-ai/d2l-en/commit/3e9358109ce349efb2fd6041f3cc496d4bc1eab2).

You need to go to the /img directory and rename the file `ChainNet.svg` to `chain-net1.svg` for this to work.
Simply a `git mv ChainNet.svg chain-net1.svg` should work for renaming.